intellij-plugin-structure/structure-teamcity-actions/src/main/kotlin/com/jetbrains/plugin/structure/teamcity/action/problems/TeamCityActionProblems.kt

```diff
@@ -0,0 +1,57 @@
+package com.jetbrains.plugin.structure.teamcity.action.problems
+
+import com.jetbrains.plugin.structure.base.problems.PluginProblem
+
+object ParseYamlProblem : PluginProblem() {
+  override val level: Level = Level.ERROR
+  override val message = "The action description should follow valid YAML syntax."
+}
+
+abstract class InvalidPropertyProblem : PluginProblem() {
+  override val level: Level = Level.ERROR
+}
+
+class InvalidPropertyValueProblem(override val message: String) : InvalidPropertyProblem()
+
+class MissingValueProblem(propertyName: String, propertyDescription: String) : InvalidPropertyProblem() {
```

Comment:
@novotnyr Hi and sorry for the late reply! I think it would be better not to use classes that inherit `InvalidDescriptorProblem` here, since the "Invalid plugin descriptor" message would be confusing to the user who is uploading actions. It's also easier for us to maintain custom `PluginProblem` classes as it makes it possible to have almost identical code in plugin verifier and TeamCity itself.